src/generateKarate.py

```diff
@@ -83,9 +88,15 @@ def processAllOfInSchema(items):
 def processOneOfInSchema(items):
     """
     This should take the contents of a OneOf and return some valid Karate matcher
-    TODO: make this work
     """
-    return {'Kungfu error': 'schema type: oneOf'}
+    karate = {}
+    for schema in items:
+        if 'properties' in schema:
+            for prop in schema['properties']:
+                karate[prop] = getKarateType(schema['properties'][prop])
+        elif 'type' in schema:
+            karate[schema['type']] = getKarateType(schema['type'])
+    return karate
```

Comment:
Looks pretty good, but in what way is this a "oneOf" handler as opposed to an "allOf" handler?

Reply:
@alxjns allOf has to handle descriptions with one more nested iteration. We can demo/discuss with @PartTimePlayer as he did a little more review with a option I sent him late yesterday.